create-aleo-app/template-vanilla/worker.js

```diff
@@ -1,47 +1,94 @@
 import {
     Account,
-    initThreadPool,
-    PrivateKey,
     ProgramManager,
-} from "@aleohq/sdk";
-
-await initThreadPool();
-
-const hello_hello_program =`
-program hello_hello.aleo;
-
-function hello:
-    input r0 as u32.public;
-    input r1 as u32.private;
-    add r0 r1 into r2;
-    output r2 as u32.private;`
-
-async function localProgramExecution(program, aleoFunction, inputs) {
-    const programManager = new ProgramManager();
-
-    // Create a temporary account for the execution of the program
-    const account = new Account();
-    programManager.setAccount(account);
-
-    const executionResponse = await programManager.run(
-        program,
-        aleoFunction,
-        inputs,
-        false,
+    PrivateKey,
+    initThreadPool,
+    AleoKeyProvider,
+    AleoNetworkClient,
+    NetworkRecordProvider,
+  } from "@aleohq/sdk";
+  
+  await initThreadPool();
+  
+  const hello_hello_program =
+      "program hello_hello_123.aleo;\n" +
+      "\n" +
+      "function hello:\n" +
+      "    input r0 as u32.public;\n" +
+      "    input r1 as u32.private;\n" +
+      "    add r0 r1 into r2;\n" +
+      "    output r2 as u32.private;\n";
+  
+  async function localProgramExecution(program, aleoFunction, inputs) {
+      const programManager = new ProgramManager();
+  
+      // Create a temporary account for the execution of the program
+      const account = new Account();
+      programManager.setAccount(account);
+  
+      const executionResponse = await programManager.run(
+          hello_hello_program,
```

Comment:
Why this change?

Reply:
The function `executeOffline` has been renamed to "run" with [this commit](https://github.com/AleoHQ/sdk/commit/45ac2ad7645499e8efbf3e76e421e4233308c9dd#diff-d5500ae2c32e9410d2a9a44bb20b6daecae531f19c3a01e1e9c8e3c90e5f21bb). That is the reason for the change.

Reply:
What I mean is, the old code passed it in as arguments, but now you're hardcoding it.

Reply:
Yes, that was a mistake I didn't realize. It seems I was too focused on deploying functionality. I've corrected it now